docs/_articles/en/einstein/einstein-inline.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Automatic Inline Completions
+lang: en
+---
+
+## Overview
+
+Use AI-based autocomplete to accept suggestions for code as you write it right inside your editor. Inline autocompletions can currently be triggered in Apex and LWC (Javascript, CSS and HTML) files.
+
+![Inline Autocomplete](./images/einstein-inline-autocomplete.gif)
+
+## Enable or Disable Inline Autocomplete
+
+Inline autocomplete is enabled by default. Click the Einstein icon in the status bar, or run **Einstein: Toggle Einstein Auto Completions** to toggle the feature off and on. Run the **Einstein: Toggle Einstein Auto Completions for Current File Type** command to toggle the feature off and on for the currently active specific file type.
+
+To enable or disable the Inline Autocomplete feature from Settings:
+
+1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
+2. Under **Einstein for Developers**, select **Enable Einstein Auto Completions** then select the language for which to enable the feature.
+
+
+
+## Receive your First Inline Completions
+
+Einstein for Developers automatically generates code and suggests completions for you as you type. When writing Apex code, you're often doing things like assigning variable values to an `Sobject` such as `Account.name = abc`, or `Account.type = Business`. Pause the cursor after the variable value and watch Einstein for Developers complete your code for you and fill out the remaining fields.
+
+![Inline Suggestions](./images/einstein-inline-create-account.png)
+
+To accept an entire suggestion, press Tab. If you have VS Code's **Inline Suggest** enabled, you can accept the next word of the suggestion using one of the following keyboard shortcuts:
+
+| Operating System | Accept Next Word |
+| ---------------- | ---------------- |
+| macOS            | ⌘→               |
+| Windows          | ⌥→               |
+| Linux            | ⌥→               |
```

Comment:
Should we doc shortcuts or just give users direction on how to see them in VSCode? Command Pallet -> Keyboard shortcuts -> Search Einstein

Reply:
IMO both just so they don't have to navigate to 2 places the first time they're looking for it in the docs